prime-router/docs/openapi.yml

```diff
@@ -1,4 +1,28 @@
 openapi: 3.0.2
+# ,---. .                                           .               
```

Comment:
I would like to see this file point to the others if possible.  E.g. it would show the RS APIs as a whole.